modules/ROOT/pages/database/database-connector-xml-maven.adoc

```diff
@@ -0,0 +1,92 @@
+= Database XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+// Add transition if needed
+
+If you manually code a Mule application in XML either from the Studio XML editor
+or a text editor, you must perform the following actions to get
+access to a connector:
+
+. Add the namespace for the connector to your XML flow.
+. Add dependency information to your Maven pom.xml file.
```

Comment:
Please check the formatting from here on out. It was wrong in the templates. It has been fixed but doesn't help you. all filenames need to be in backticks. For example, `pom.xml`.

Reply:
Added backticks.